build-farm/make-adopt-build-farm.sh

```diff
@@ -252,4 +252,9 @@ echo "$MAC_ROSETTA_PREFIX $PLATFORM_SCRIPT_DIR/../makejdk-any-platform.sh --clea
 CONFIGURE_ARGS_FOR_ANY_PLATFORM="${CONFIGURE_ARGS_FOR_ANY_PLATFORM//\"/temporary_speech_mark_placeholder}"
 
 # shellcheck disable=SC2086
-bash -c "$MAC_ROSETTA_PREFIX $PLATFORM_SCRIPT_DIR/../makejdk-any-platform.sh --clean-git-repo --jdk-boot-dir ${JDK_BOOT_DIR} --configure-args \"${CONFIGURE_ARGS_FOR_ANY_PLATFORM}\" --target-file-name ${FILENAME} ${TAG_OPTION} ${OPTIONS} ${BUILD_ARGS} ${VARIANT_ARG} ${JAVA_TO_BUILD}"
\ No newline at end of file
+bash -c "$MAC_ROSETTA_PREFIX $PLATFORM_SCRIPT_DIR/../makejdk-any-platform.sh --clean-git-repo --jdk-boot-dir ${JDK_BOOT_DIR} --configure-args \"${CONFIGURE_ARGS_FOR_ANY_PLATFORM}\" --target-file-name ${FILENAME} ${TAG_OPTION} ${OPTIONS} ${BUILD_ARGS} ${VARIANT_ARG} ${JAVA_TO_BUILD}"
+
+if [ -d "${WORKSPACE}" ]; then
+  SPACEUSED=$(du -sk "$WORKSPACE")
+  echo "Total disk space in Kb consumed by build process: $SPACEUSED"
```

Comment:
Could we also dump the current volume's free disk space? This could help figure the github runner issue?

Reply:
Could we also print this info at the start of the build as well?

Reply:
I considered that but I'd rather not do that in this PR because the before and after data is potentially misleading for docker containers which use shared space, plus we have Nagios monitoring to detect problems in that area
2. For the github runner case, presumably you could do a subsequent PR with the change you want - that would allow you to test it in the GHA checks even if you don't end up merging it.

Reply:
yes makes sense